e2e/config/config.go

```diff
@@ -15,7 +15,7 @@ var (
 	Azure             = mustNewAzureClient()
 	ResourceGroupName = "abe2e-" + Config.Location
 	VMIdentityName    = "abe2e-vm-identity"
-	PrivateACRName    = "privateacre2e" + Config.Location
+	PrivateACRName    = "xinhlprivateacre2e" + Config.Location
```

Comment:
im assuming this is just for testing atm

Reply:
the name is for temp to avoid conflict with old tests